site/src/analytics-integrations.ts

```diff
@@ -124,6 +177,50 @@ export class AnalyticsIntegrations extends LitElement {
     `;
   }
 
+  private _computeName(entry: IntegrationData): string {
+    return (
+      this._integrationDetails[entry.integration]?.title || entry.integration
+    );
+  }
+
+  private _filterChange(ev: any) {
+    this._currentTablePage = 0;
+    this._filter = ev.currentTarget?.value || "";
+  }
+
+  private _clearFilter() {
+    this._currentTablePage = 0;
+    this._filter = "";
+  }
+
+  async getData() {
+    try {
+      const response = await ((window as any).integrationsPromise ||
+        fetchIntegrationDetails());
```

Comment:
What could cause that `integrationsPromise` is not defined?

Reply:
Time, the same approach is currently used for the worker data